### DISCUSSION TITLE
Customizable syntax highlighting

### DISCUSSION THREAD

**Discussion by heavyelement** [mdn/mdn-community#299] [MDN Plus] [answered]
MDN Plus feature request: customizable colors for syntax highlighting on the docs. I'd love to be able to set the colors to match my IDE.

Or even if there were different schemes that matched popular IDEs, it'd be sick.

**Comment by schalkneethling** [comment, marked as answer]
> MDN Plus feature request: customizable colors for syntax highlighting on the docs. I'd love to be able to set the colors to match my IDE.
> 
> Or even if there were different schemes that matched popular IDEs, it'd be sick.

Hey @heavyelement, thank you for your suggestion. We will definitely take this under consideration. If this is shipped, we will be sure to let you know. Keep the ideas coming.
